### DISCUSSION TITLE
Mega Menu vertical - is it possible for the panel to open to the left?

### DISCUSSION THREAD

**Discussion by BySAQQ** [primefaces/community#197] [PrimeNG] [answered]
<p-megaMenu [model]="items" orientation="vertical"></p-megaMenu>

I have this megaMenu and I want the items to be displayed from right to left. Is there a way to do it ?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, MegaMenu doesn't have this as a feature.

**Reply by BySAQQ** [reply]
thanks for replying, @cetincakiroglu. 😎
I will search for another option

**Comment by cagataycivici** [comment]
We'll improve the RTL support for the whole suite in the near future.
